app/javascript/src/components/Dashboard/Notes/NoteList/index.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+
+import Note from "./Note";
+
+import DeleteAlert from "../DeleteAlert";
+import EditNotePane from "../Pane/Edit";
+
+const NoteList = ({ notes = [], fetchNotes }) => {
+  const [showEditNote, setShowEditNote] = useState(false);
+  const [selectedNote, setSelectedNote] = useState({});
+  const [showDeleteAlert, setShowDeleteAlert] = useState(false);
+  const [selectedDeleteNoteIds, setSelectedDeleteNoteIds] = useState([]);
+  return (
+    <>
+      <div>
+        {notes.map((note, idx) => (
+          <Note
+            key={idx}
```

Comment:
https://academy.bigbinary.com/learn-react/pages/using-unique-keys-for-list-elements#:~:text=Don%E2%80%99t%20use%20the%20index%20as%20a%20key

Go through this chapter very well. The point of writing something in the React Best Practices book is to ensure that we avoid review comments like this. So make sure to go through React best practices(no ramda docs needed)

Reply:
Pardon me Yedhin for that. It came as a force of habit. I have corrected the mistake by utilizing uuids.